app/src/org/commcare/views/QuestionsView.java

```diff
@@ -65,12 +66,14 @@
 
     private SpannableStringBuilder mGroupLabel;
 
+    private BlockingActionsManager blockingActionsManager;
+
     /**
      * If enabled, we use dividers between question prompts
      */
     private static final boolean SEPERATORS_ENABLED = false;
 
-    public QuestionsView(Context context) {
+    public QuestionsView(Context context, BlockingActionsManager blockingActionsManager) {
```

Comment:
Is there a reason we pass the blocking actions manager down the chain directly v. adding it to the widget factory? It seems like the widget factory is a cleaner place to keep the refrence